Software-Engineering.md

```diff
@@ -133,6 +133,29 @@ _We avoid distinguishing too much between the Junior Software Engineer and Softw
 - I can confidently and charismatically pitch ideas, positively influencing my team to take decisive action based on reaching consensus.
 - I am an adept communicator, and can effectively steer technical and non-technical conversations to positive outcomes over any medium.
 
+## Principal Software Engineer
+
+> I am an influential Engineering [leader](Leadership.md) who identifies problems and opportunities relevant to Octopus' strategy, builds consensus to invest in solutions, and guides Engineering to deliver significant lasting change across Octopus.
+
+### I uncover important problems and opportunities that are critical to the long-term success of Octopus
+
+- I have a deep understanding of Octopus products and strategy
+- I collaborate with other Engineering leaders to ensure I am aware of all major initiatives at Octopus and can account for them in my own initiatives
+- I have a track record of identifying significant problems, inhibitors and opportunities, and for creating lasting, best-fit solutions that align our strategy and Engineering efforts
+- I work closely with product, engineering, and executive stakeholders to articulate and refine solutions to problems that are timely, important, and impactful. 
+- My contributions go beyond patching or extending existing solutions, which may not be fit for purpose anymore. They open up whole new areas of strategic possibility that progress our journey of making Octopus a world-class Engineering organization.
```

Comment:
"They" in the second sentence caused me to pause for a moment due to ambiguity. Is this "they" the person... oh, "they" is the contribution. I've made a small suggestion to clarify:
```suggestion
- Instead of simply patching or extending existing solutions, which may not be fit for purpose anymore, my contributions open up whole new areas of strategic possibility that progress our journey of making Octopus a world-class Engineering organization.
```